bindata/network/ovn-kubernetes/common/alert-rules.yaml

```diff
@@ -26,3 +26,13 @@ spec:
       for: 20m
       labels:
         severity: warning
+   - alert: OVNKubernetesOvnKubeMasterLostConnectionToDatabases
+      annotations:
+        summary: Networking control plane is degraded on node {{"{{"}} $labels.node {{"}}"}} because OVNKubeMaster lost connections to OVN SB/NB databases.
```

Comment:
If its master loosing connection to db's then its control plane alerts. This file is for data plane alerts (ovnkube-node alerts). You should probably add this to control-plane alerts file and test it.

Reply:
will attempt testing this when all pieces in place and content make sense